src/components/Health/__tests__/HealthIndicator.test.tsx

```diff
@@ -140,6 +140,6 @@ describe('HealthIndicator', () => {
     html = wrapper.html();
     expect(html).toContain(PFAlertColor.Danger);
     expect(html).toContain('Outbound: 16.67%');
-    expect(html).toContain('Inbound: 26.47%');
+    expect(html).toContain('Inbound: 50.00%');
```

Comment:
hmm .. so there was a bug here?

Reply:
@jotak Right, the bug fix in ErrorRate.ts uncovered the bad expectation in this test.  So, a bug in the code and a bug in the test, a bad combination but it happens.